runtime/src/core/closures.js

```diff
@@ -1,17 +1,16 @@
-import { managedMemory, grainModule } from "../runtime";
 import { grainToJSVal, JSToGrainVal } from "../utils/utils";
 
 export class GrainClosure {
   constructor(loc, runtime) {
-    const view = managedMemory.view;
+    const view = runtime.managedMemory.view;
 
     this.loc = loc;
     this.runtime = runtime;
     this.arity = view[loc];
-    this.ptr = view[loc + 1];
-    this.closureSize = view[loc + 2];
-    this.closureElts = view.slice(loc + 3, loc + 3 + this.closureSize);
-    this.func = grainModule.instance.exports["GRAIN$LAM_" + this.ptr];
+    this.ptr = view[loc + 2];
```

Comment:
Was this just broken before, or did one of the changes in the compiler in this PR change the memory layout of closures?

Reply:
This was just broken. This code isn't used or exercised at all and we should consider just deleting it 😶